outages/README.md

```diff
@@ -0,0 +1,48 @@
+## How to manage outages
+
+### Who is responsible for resolving outages and what should be done
+
+- rotating role - new / incorporate to an existing one (Sentry master, community master)
```

Comment:
This might not be necessary given a well-developed monitoring and alerting. Until then, "looking for signs" could be the responsibility of the Sentry master.